Combobox.js

```diff
@@ -252,10 +252,54 @@ define([
 		 */
 		cancelMsg: messages["cancel-button-label"],
 
+		/**
+		 * Displays or not the down arrow button.
+		 * @type {boolean}
+		 * @default true
+		 */
+		hasDownArrow: true,
+
+		/**
+		 * Defines if a click on widget itself will show up the list immediately.
+		 * It makes sense only on desktop version, since in mobile, a click on the widget opens the
+		 * popup widget always.
+		 * @type {boolean}
+		 */
+		openOnPointerDown: true,
+
+		/**
+		 * Flag that allows lazy initialization of list's source.
+		 * @type {boolean}
+		 */
+		_assignSourceToList: false,
+
+		/**
+		 * Source for the inner list.
+		 * @type {delite/Store} Source set.
+		 */
+		source: null,
+
+		/**
+		 * Minimum number of characthers before a filter operation runs.
+		 * @type {Number}
+		 * @default 1
+		 */
+		minChars: 1,
+
+		/**
+		 * Initial inputNode's value.
+		 * @type {String}
+		 */
+		displayedValue: "",
+
 		createdCallback: function () {
 			// Declarative case (list specified declaratively inside the declarative Combobox)
 			var list = this.querySelector("d-list");
 			if (list) {
+				if (list.source !== null) {
+					// The widget has to get the list's source, if any.
+					this.source = list.source;
+				}
```

Comment:
We actually need that code because this is the case when the developers create a Combobox decoratively, like below:
```
<d-combobox selectionMode="multiple" id="comboPlayersWithValuesDecl" autoFilter="true" value="Hagi,Buffon">
	<d-list>
		{ "label": "Zidane" },
		{ "label": "Platini" },
		{ "label": "Maradona" },
		{ "label": "Hagi" },
		{ "label": "Buffon" }
	</d-list>
</d-combobox>
```
In this case we need to catch the list's source, so the widget has his own copy. 

Aside that we don't need to nullify the `list.source`. In fact even though the list is already initialized, it won't fire any spurious request, because the source is just a Array store (and not i.e. a `dstore/Request`).